contracts/characters.sol

```diff
@@ -24,6 +24,8 @@ contract Characters is Initializable, ERC721Upgradeable, AccessControlUpgradeabl
         __AccessControl_init_unchained();
 
         _setupRole(DEFAULT_ADMIN_ROLE, msg.sender);
+
+        characterLimit = 4;
```

Comment:
For now I don't intend to enforce it, but for the future: initialization of new state variables have to be done by a brand-new function as the initializer function was called exactly once - during the first deploy - and will never be called again.

Our convention has been to add a `migrateTo_XXXXXXX` function that gets called during deployment of the new version of the contract, where `XXXXXXX` is the 7 first characters of some relevant Git commit hash that introduced the changes. You can find examples by looking in several other files.

Do feel free to implement that if you wish, but once again, I don't intend to enforce it for now.